examples/cleanrl_examples/ppo_atari_envpool.py

```diff
@@ -375,8 +389,12 @@ def get_action_and_value(self, x, action=None):
       ).to(device)
 
       for idx, d in enumerate(done):
-        if d:
+        if d and info["lives"][idx] == 0:
```

Comment:
@vwxyzjn I also don't understand why we are checking that all lives have been exhausted `info["lives"][idx] == 0:` although the episode may not be over (lives > 0). I have also looked over all the cleanrl examples, and this condition is only being checked for cleanrl examples that are for envpool (e.g. the dqn, c51 atari examples don't check for lives, although they use the standard gym RecordEpisodeStatistics wrapper)